src/components/_modal/pages/bookmarkOrder/multi-order/Material/Material.js

```diff
@@ -173,13 +171,15 @@ const Material = ({ material, setMaterialsToOrder, context }) => {
       backgroundColor,
     });
 
-  const firstMoreInfoManifestation = flattenedGroupedSortedManifestations.find(
-    (manifestation) => manifestation.cover.detail.startsWith("https://moreinfo")
+  // If we have manifestations with cover, take the first one
+  const manifestationsWithCover = getManifestationWithoutDefaultCover(
+    flattenedGroupedSortedManifestations
   );
 
-  // If none found, take the first manifestation in the array
-  const firstManifestation =
-    firstMoreInfoManifestation || flattenedGroupedSortedManifestations[0];
+  // If no cover found, take the first manifestation in the array
+  const firstManifestation = !isEmpty(manifestationsWithCover)
```

Comment:
Hvis du bruger `getManifestationsWithCorrectCover()` i stedet for, så kan du ændre linje 175,176,179, 180, 181, 181 til:
```
const { manifestationsWithCover: firstManifestation } = getManifestationsWithCorrectCover(flattenedGroupedSortedManifestations)
```